packages/errors/src/errors.ts

```diff
@@ -155,25 +155,31 @@ export const AllCypressErrors = {
   CLOUD_CANCEL_SKIPPED_SPEC: () => {
     return errTemplate`${fmt.off(`\n  `)}This spec and its tests were skipped because the run has been canceled.`
   },
-  CLOUD_API_RESPONSE_FAILED_RETRYING: (arg1: {tries: number, delayMs: number, response: Error}) => {
+  CLOUD_API_RESPONSE_FAILED_RETRYING: (
+    arg1: {tries: number, delayMs: number, response: Error },
+  ) => {
     const time = pluralize('time', arg1.tries)
     const delay = humanTime.long(arg1.delayMs, false)
 
+    const message = arg1.response.name === 'AxiosError' ? arg1.response.message : `${arg1.response.name}: ${arg1.response.message}`
```

Comment:
This logic is duplicated several times. Worth DRYing up?

```suggestion
    const message = arg1.response.name === 'AxiosError' ? arg1.response.message : `${arg1.response.name}: ${arg1.response.message}`
```